### DISCUSSION TITLE
Teacher looking to use StreamDeck to assist in the classroom

### DISCUSSION THREAD

**Discussion by MathZombie12** [elgatosf/elgato#66] [Help]
I'm a teacher and want to use the Stream Deck to help with some classroom stuff. I currently have it set up with some sound cues and some timers on the buttons. 

What I really would like is to push the 5 minute timer button and have a 5 minute countdown timer appear on my screen or on top of my google slides presentation. I don't know if that's manageable or difficult to do. I run it on a MacBook Air. Any help would be greatly appreciated. Thank you!

(For example, often in class I'm running a Google Slides presentation where I display problems or activities for the students. I'd love to be able to push a button on the stream deck that would pop a 5 minute timer in the corner of the screen that counts down so the kids can see it.)

**Comment by ChekTek** [comment]
Hi!

You may be able to accomplish this with the Head-up-Display plugin, but unfortunately this is Windows only.

GitHub discussions are primarily for developers working with the Stream Deck software development kit. If you are looking for general support or feature requests, please check out [r/elgato](https://www.reddit.com/r/elgato/) or http://e.lga.to/help.
